res/lang/gen_settings/genSettings_en.txt

```diff
@@ -109,20 +109,20 @@ TWO_DAYS_AGO = Two days ago
 IN_X_DAYS    = In %DAYS% days
 X_DAYS_AGO   = %DAYS% days ago
 
-WEEK_SHORT_MONDAY    = mo
-WEEK_SHORT_TUESDAY   = tu
-WEEK_SHORT_WEDNESDAY = we
-WEEK_SHORT_THURSDAY  = th
-WEEK_SHORT_FRIDAY    = fr
-WEEK_SHORT_SATURDAY  = sa
-WEEK_SHORT_SUNDAY    = su
-WEEK_LONG_MONDAY     = monday
-WEEK_LONG_TUESDAY    = tuesday
-WEEK_LONG_WEDNESDAY  = wednesday
-WEEK_LONG_THURSDAY   = thursday
-WEEK_LONG_FRIDAY     = friday
-WEEK_LONG_SATURDAY   = saturday
-WEEK_LONG_SUNDAY     = sunday
+WEEK_SHORT_MONDAY    = Mo
```

Comment:
Hmm wenn man die am Satzanfang will, "ucfirst". Ansonsten ist zumindest die "Langform" nun nicht mehr mittels

The next possible air date is this %WEEKDAY_LONG%.

Reply:
Wochentage und Monate sind im Englischen doch aber immer groß geschriebene Eigennamen...

Reply:
Stimmt .. irgendwie ... ist mir das entfallen - aber Nachlesen half.